lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -221,4 +224,12 @@ module.exports = {
 
     return auth;
   },
+
+  validateAlbHealthCheck(event) {
+    const eventHealthCheck = event.alb.healthCheck;
+    if (_.isObject(eventHealthCheck)) {
+      return Object.assign(eventHealthCheck, { enabled: true });
```

Comment:
Let's reverse arguments order (as otherwise we're translating `{ enabled: false }` into `{ enabled: true }` and that's bug)

Reply:
That was by design in order to avoid an creating an additional `heallthCheck.enabled` property that end users could set in `serverless.yml`. Reversing the order would make the following possible:

```yaml
alb:
  healthCheck:
    enabled: false
```

Reply:
Indeed. I forgot we agreed to not support `enabled` property, as it hardly makes sense.

Sorry for confusion